packages/server/modules/shared/test/unit/eventBus.spec.ts

```diff
@@ -106,69 +87,55 @@ describe('Event Bus', () => {
       const bus1 = getEventBus()
       const bus2 = getEventBus()
 
-      const workspaces: Workspace[] = []
+      const payloads: string[] = []
 
-      bus1.listen(WorkspaceEvents.Created, ({ payload }) => {
-        workspaces.push(payload)
+      bus1.listen('test.string', ({ payload }) => {
+        payloads.push(payload)
       })
 
-      bus2.listen(WorkspaceEvents.Created, ({ payload }) => {
-        workspaces.push(payload)
+      bus2.listen('test.string', ({ payload }) => {
+        payloads.push(payload)
       })
 
-      const workspacePayload = {
-        ...createFakeWorkspace(),
-        createdByUserId: cryptoRandomString({ length: 10 }),
-        eventName: WorkspaceEvents.Created,
-        domains: []
-      }
+      const payload = cryptoRandomString({ length: 1 })
 
       await bus1.emit({
-        eventName: WorkspaceEvents.Created,
-        payload: { ...workspacePayload }
+        eventName: 'test.string',
+        payload
       })
 
-      expect(workspaces.length).to.equal(2)
-      expect(workspaces).to.deep.equal([workspacePayload, workspacePayload])
+      expect(payloads.length).to.equal(2)
+      expect(payloads).to.deep.equal([payload, payload])
     })
     it('allows to subscribe to wildcard events', async () => {
       const eventBus = getEventBus()
 
       const events: string[] = []
 
-      eventBus.listen('workspace.*', ({ payload, eventName }) => {
+      eventBus.listen('test.*', ({ payload, eventName }) => {
         switch (eventName) {
-          case 'workspace.created':
-            events.push(payload.id)
+          case 'test.string':
```

Comment:
can we use the enums here? it makes finding all usages easier, i was already running into this problem in my feature branch when looking through workspace events for subscriptions